types/qunit/qunit-tests.ts

```diff
@@ -214,13 +214,46 @@ QUnit.log(function( details ) {
   console.log( output );
 });
 
+QUnit.log(function( details ) {
+  const aDetails: QUnitLogDetails = details;
```

Comment:
Is it necessary to create this extra variable?

Reply:
Also not sure all these changes to the tests are necessary, what are you trying to test?

Reply:
This assignment is necessary in order to assert that callbacks passed to `QUnit.log` receive an argument that is type-equivalent to a `QUnitLogDetails` type. Everything else in the test simply validates that individual properties exist (not even that they're of the correct type).